postgres/server/init.sls

```diff
@@ -217,20 +234,65 @@ postgresql-tablespace-dir-{{ name }}:
     - require:
       - pkg: postgresql-server
 
+    {%- if "selinux" in grains and grains.selinux.enabled %}
+
+  pkg.installed:
+    - names:
+      - policycoreutils-python
+      - selinux-policy-targeted
+    - refresh: True
+  selinux.fcontext_policy_present:
+    - name: '{{ tblspace.directory }}(/.*)?'
+    - sel_type: postgresql_db_t
+    - require:
+      - file: postgresql-tablespace-dir-{{ name }}
+      - pkg: postgresql-tablespace-dir-{{ name }}
+
+postgresql-tablespace-dir-{{ name }}-fcontext:
+  selinux.fcontext_policy_applied:
+    - name: {{ tblspace.directory }}
+    - recursive: True
+    - require:
+      - selinux: postgresql-tablespace-dir-{{ name }}
+
+    {%- endif %}
+
 {%- endfor %}
 
 {%- if not postgres.bake_image %}
 
+# Workaround for FreeBSD minion undefinitely hanging on service start
+# cf. https://github.com/saltstack/salt/issues/44848
+{% if postgres.service.sysrc %}
+posgresql-rc-flags:
+  sysrc.managed:
+    - name: {{ postgres.service.name }}_flags
+    - value: "{{ postgres.service.flags }} > /dev/null 2>&1"
+    - watch_in:
+      - service: postgresql-running
+{% endif %}
+
 # Start PostgreSQL server using OS init
+# Note: This is also the target for numerous `watch_in` requisites above, used
+# for the necessary service restart after changing the relevant configuration files
 postgresql-running:
   service.running:
-    - name: {{ postgres.service }}
+    - name: {{ postgres.service.name }}
     - enable: True
-   {% if grains.os not in ('MacOS',) and not postgres.restart_service_on_changes %}
```

Comment:
@EliAndrewC Can you review this line and below? This is where the merge conflict was, and I want to make sure we aren't accidentally undoing part of https://github.com/magfest/postgres-formula/commit/b3df53ff55fde3dbebaac064b1935b1d5adea445 with the way I resolved it. The current state of upstream master is here: https://github.com/saltstack-formulas/postgres-formula/blob/master/postgres/server/init.sls#L275

Reply:
Sorry it took me so long to get around to reviewing this.

This is fine to remove - the equivalent functionality has been added above by this:

```
    - watch_in:
￼	      - service: postgresql-running
```

which is the preferred way to handle this.  This should be fine to merge.